docs/content/latest/api/ysql/the-sql-language/statements/ddl_create_index.md

```diff
@@ -47,9 +47,57 @@ Use the `CREATE INDEX` statement to create an index on the specified columns of
 
 `CONCURRENTLY`, `USING method`, `COLLATE`, and `TABLESPACE` options are not yet supported.
 
+When an index is created on a populated table, YugabyteDB automatically
+backfills the existing data into the index.  In most cases, this uses an online
+schema migration.
+
+| | online | not online |
+| --- | --- | --- |
+| safe to do other DMLs during `CREATE INDEX`? | yes\* | no |
+| keeps other transactions alive during `CREATE INDEX`? | mostly | no |
+| parallelizes index loading? | yes | no |
+
+To disable online schema migration for YSQL `CREATE INDEX`, set the flag
+`ysql_disable_index_backfill=true` on **all** nodes and **both** master and
+tserver.  To disable online schema migration for one `CREATE INDEX`, use
+`CREATE INDEX NONCONCURRENTLY`.
+
+If online `CREATE INDEX` fails, it likely failed in the backfill step.  In that
+case, the index exists but is not usable.  Drop the index and try again.  If it
+still doesn't work,
+
+- did it time out?  Then, try increasing timeout flags:
+  - master `ysql_index_backfill_rpc_timeout_ms`
+  - tserver `ysql_wait_until_index_permissions_timeout_ms`
+- did you get "backfill failed to connect to DB"?  Then, you may be hitting an
+  issue with authentication.  If the version is <2.4 (stable) or <2.5.2
+  (latest), online `CREATE INDEX` does not work with authentication enabled.
+  - If the version is 2.5.1 (latest), then you can use `CREATE INDEX
+    NONCONCURRENTLY` as a workaround.
+  - If the version is >=2.3 (latest), then you can set
+    `ysql_disable_index_backfill=false` as a workaround.
+  - In any case, you can disable authentication (e.g. using `ysql_enable_auth`,
+    `ysql_hba_conf`, or `ysql_hba_conf_csv`) as a workaround.
+- did you get "duplicate key value" error?  Then, you have a unique constraint
+  violation.
+
+Please file a GitHub issue if you still have issues.
```

Comment:
Let's direct to Slack first, and GitHub second. Likely a faster response on Slack.
```suggestion
Please ask for help in our [community Slack](https://www.yugabyte.com/slack) or [file a GitHub issue](https://github.com/yugabyte/yugabyte-db/issues/new?title=Index+backfill+failure) if you still have issues.
```